AlphaWallet/UI/AddressTextField.swift

```diff
@@ -245,6 +245,7 @@ class AddressTextField: UIControl {
         scanQRCodeButton.translatesAutoresizingMaskIntoConstraints = false
         scanQRCodeButton.setImage(R.image.qr_code_icon(), for: .normal)
         scanQRCodeButton.addTarget(self, action: #selector(openReader), for: .touchUpInside)
+        scanQRCodeButton.backgroundColor = .clear
```

Comment:
I thought setting `backgroundColor` on `UIButton` isn't so predictable. Is it necessary to fix this?

Reply:
> I thought setting `backgroundColor` on `UIButton` isn't so predictable. Is it necessary to fix this?

i suppose we can replace it with adding new case for ButtonStyle, (but behind the scene it still will bet background color value for UIButton)